src/Shared/CertificateGeneration/MacOSCertificateManager.cs

```diff
@@ -375,7 +375,7 @@ protected override IList<X509Certificate2> GetCertificatesToRemove(StoreName sto
 
     protected override void PopulateCertificatesFromStore(X509Store store, List<X509Certificate2> certificates)
     {
-        if (store.Name! == StoreName.My.ToString() && store.Location == store.Location && Directory.Exists(MacOSUserHttpsCertificateLocation))
```

Comment:
So, did store.Location ever not equal store.Location previously?

Reply:
No, but it happens to be harmless since the alternative to `CurrentUser` is `LocalMachine`, and in that case, we'll basically end up doing the same thing as the `else` path (i.e. load the certs from that store), since we'll try to also find matching certs on disk and not find any. The funny/sad thing is, I had already fixed this typo during the original PR but it snuck back in as part of the late changes and the cross-commit merges.